docs/zh/latest/FAQ.md

```diff
@@ -21,73 +21,97 @@ title: 常见问题
 #
 -->
 
-## 为什么要做 API 网关？不是已经有其他的开源网关了吗？
+## 为什么我们需要一个新的 API 网关？不是已经有其他的开源网关了吗？
 
-微服务领域对 API 网关有新的需求：更高的灵活性、更高的性能要求，以及云原生的贴合。
+随着企业向云本地微服务的发展，对高性能、灵活、安全、可扩展的API网关的需求越来越大，一些之前的网关已经逐渐无法满足当今企业的需求。
+
+对于上面所述的这些特点来说，APISIX相对于其它网关要做的更好，同时APISIX还具有平台无关性和完全动态交付特性，比如支持多协议、细粒度路由和多语言支持。
 
 ## APISIX 和其他的 API 网关有什么不同之处？
 
-APISIX 基于 etcd 来完成配置的保存和同步，而不是 PostgreSQL 或者 MySQL 这类关系型数据库。
-这样不仅去掉了轮询，让代码更加的简洁，配置同步也更加实时。同时系统也不会存在单点，可用性更高。
+Apache APISIX的不同之处在于:
```

Comment:
```suggestion
Apache APISIX在以下方面有所不同：
```

Reply:
@guoqqqi 
Ok, I will pull the request again today.